From 4ccea72e2f2150f78b25beed766c0dd140d5fd82 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:13 -0400
Subject: [usb] xhci: Staticize xhci_del_comp_mod_timer

Message-id: <1410887463-20674-62-git-send-email-dzickus@redhat.com>
Patchwork-id: 95202
O-Subject: [RHEL7 PATCH 061/591] usb: xhci: Staticize xhci_del_comp_mod_timer
Bugzilla: 1032371
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1032371

commit 5f20cf12a63650afe1871c7b2c89ee84ec3c6182
Author: Sachin Kamat <sachin.kamat@linaro.org>
Date:   Mon Sep 16 12:01:34 2013 +0530

    usb: xhci: Staticize xhci_del_comp_mod_timer

    'xhci_del_comp_mod_timer' is local to this file.

    Signed-off-by: Sachin Kamat <sachin.kamat@linaro.org>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci-hub.c b/drivers/usb/host/xhci-hub.c
index e8b4c56..45ca532 100644
--- a/drivers/usb/host/xhci-hub.c
+++ b/drivers/usb/host/xhci-hub.c
@@ -524,7 +524,8 @@ static void xhci_hub_report_usb3_link_state(u32 *status, u32 status_reg)
  * the compliance mode timer is deleted. A port won't enter
  * compliance mode if it has previously entered U0.
  */
-void xhci_del_comp_mod_timer(struct xhci_hcd *xhci, u32 status, u16 wIndex)
+static void xhci_del_comp_mod_timer(struct xhci_hcd *xhci, u32 status,
+        u16 wIndex)
 {
  u32 all_ports_seen_u0 = ((1 << xhci->num_usb3_ports)-1);
  bool port_in_u0 = ((status & PORT_PLS_MASK) == XDEV_U0);
-- 
1.7.1